python/sglang/srt/managers/io_struct.py

```diff
@@ -57,9 +57,7 @@ class GenerateReqInput:
     lora_path: Optional[Union[List[Optional[str]], Optional[str]]] = None
 
     def normalize_batch_and_arguments(self):
-        if (self.text is None and self.input_ids is None) or (
-            self.text is not None and self.input_ids is not None
-        ):
+        if self.text is None and self.input_ids is None:
```

Comment:
I do not think you can change this. Only one of them can be provided. Otherwise, it is ambiguous.